dotcom-rendering/src/web/components/PinnedPostLiveness.importable.tsx

```diff
@@ -25,9 +27,31 @@ export const PinnedPostLiveness = ({}) => {
 		pinnedPost && pinnedPost.scrollHeight <= pinnedPost.clientHeight;
 	if (contentFitsContainer) hideShowMore();
 
+	const hasBeenSeen = useRef(false);
+
 	useEffect(() => {
 		const handleClickTracking = () => {
-			// TODO: ADD OPHAN CLICK TRACKING
+			if (pinnedPostCheckBox instanceof HTMLInputElement) {
+				if (pinnedPostCheckBox.checked) {
+					submitComponentEvent({
+						component: {
+							componentType: 'LIVE_BLOG_PINNED_POST',
+							id: pinnedPost?.id,
+						},
+						action: 'CLICK',
+						value: 'show-less',
```

Comment:
Just checking the `value` field and I wonder if these are coming out the wrong way around? When `checked` is true the click event should have been `show-more`, but it's currently `show-less`  does that sound correct?

Reply:
Yeaa you're right. I'm just a bit confused as why we have it this way in frontend? Maybe that also needs to change? https://github.com/guardian/frontend/blob/c19ce5ce2f0bfe48868e9a5527a2c40f96e459e8/static/src/javascripts/bootstraps/enhanced/liveblog.js#L141

Reply:
Yeah good question - I think @abeddow91 added those to Frontend so she might be the person to ask!

Reply:
Yep this certainly does look to be the wrong way round. I'll pull this branch and have a play. In Frontend, the box is checked before the event fires which is why they seem to be the wrong way round but it would be good to check if this is the same here.

Reply:
This is the wrong way round here. I've committed the change whilst Marjan's on AL